models/marts/rollups/agg_product_cat_monthly.sql

```diff
@@ -0,0 +1,118 @@
+with
+
+orders as (
+
+    select 
+        *,
+        timestamp_trunc(reviewed_at, month) as date_month 
```

Comment:
More traditionally, these imports would not include new columns being declared, however as this is being used a few times it may make sense to define it up top. Be that in this CTE or another, both are good ideas because then you don't have to write this line 3 times below

Reply:
noted, next time in the future I will create a cte that derives this additional dim